ygen/goelements.go

```diff
@@ -465,6 +465,9 @@ func (s *genState) buildListKey(e *yang.Entry, compressOCPaths bool) (*YangListA
 	var errs []error
 	keys := strings.Split(e.Key, " ")
 	for _, k := range keys {
+		if k == "" {
+			continue
+		}
```

Comment:
I think this doesn't handle tabs or newline:
```
line-break          = CRLF / LF
WSP                 = SP / HTAB
                         ; whitespace
```
I didn't figure out how to deal with CRLF, but maybe [strings.Fields](https://golang.org/pkg/strings/#Fields), which uses [unicode.IsSpace](https://golang.org/pkg/unicode/#IsSpace) as the separator, is sufficient.

Reply:
Good catch -- let me look in goyang -- I've never seen a CRLF in this field - I'm not sure whether goyang does anything to it before it hands it downstream.

Reply:
It looks like goyang removes tabs, but keeps newlines when it parses a key string with these whitespaces. So, I think `strings.Fields` is an appropriate function to use. See https://github.com/openconfig/goyang/pull/107